R/UnsupervisedTask.R

```diff
@@ -1,6 +1,6 @@
 makeUnsupervisedTask = function(type, data, weights = NULL, blocking = NULL, fixup.data = "warn", check.data = TRUE) {
   task = makeTask(type, data, weights, blocking, fixup.data = fixup.data, check.data = check.data)
-  checkTaskData(getTaskData(data))
+  checkTaskData(task$env$data)
```

Comment:
>wouldn't have worked otherwise
is pretty vague. And as I am the second person to ask maybe a comment in the source would be usefull. But are we sure that `checkTaskData(getTaskData(task))` does not work? What would be the error?